bindings/bind_py_module.cc

```diff
@@ -50,9 +53,9 @@ PYBIND11_MODULE(_rascal, mod) {
 
```

Comment:
sorry again not part of the pull request, but for what reason there exists this rubbish package, is it to test functionalities? Some more explanation would be helpful

Reply:
no it is rubbish the proper naming. it has all the stuff that is binded, used but the user should not know about.

Reply:
okay, can we add a bit more documentation to it
>
  py::module m_throwaway = mod.def_submodule("rubbish");
  m_throwaway.doc() =
      "Collection of bindings that are needed to build functional bindings for"
      " the python user but are not functional itself. It basically contains"
      " all methods which are used on the binding side, but are not meant to"
      " be used by the python user. It is also not part of the rascal library";

Reply:
... can we give this a better name? rubbish is something that should be quickly disposed of. if this is meant to stay, and is just a collection of internal utility functions, would something like "_utilities" or "_internal" module name be more appropriate?

Reply:
do as you feel is most appropriate.

Reply:
We can follow lolcode conventions and call it "invisible"

Reply:
Ok, to pick this up again: The stuff in this module is persistent, but just not public? And it's basically internals/utils/whatever? Then the underscore-naming convention would seem to be appropriate, e.g. `_utils` or `_internals`, as @ceriottm suggested.